src/components/@core/navigation-menu/light/left-menu/logo.tsx

```diff
@@ -82,6 +91,7 @@ export default function PrimaryLogo({ isOpen, onToggle }) {
         <Icon name={isOpen ? "ibpcross" : "ibpmenu"} />
       </button>
       {isLoggedIn && id && <JoinUserGroup />}
+      {canEdit() && <EditLinkButton label={t("GROUP.EDIT.TITLE")} webAddress={webAddress} />}
```

Comment:
Instead of calling function here you can use `useState` also it will be better if you can extract this into seprate component